src/express/controllers/alt-leaderboard.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+/*flow-include
+import type { Request, Response } from 'express';
+import type { State } from '../../redux/store.js';
+*/
+
+
+const altLeaderBoardCtrl = (req /*: Request */, res /*: Response */) => {
+  const store = req.app.get('store');
+  const state /*: State */ = store.getState();
+  const { games } = state;
+
+  const countAllGamesPerPlayer = [].concat.apply([],
+    games
+    .map(game => game.rankings)
+    .map(game => Object.keys(game))
+  ).reduce(reduceCount, {});
```

Comment:
Si t'as un pattern plus clean pour rendre flat, why not (j'avais la flemme de boucler et passant par une autre variable).